### PR TITLE
feat: add to calendar links in ticket purchaser mail

### LINKED ISSUE #8280: Add to calendar option in email for attendees
Add line "Add to calendar" to order email for attendees.

1. We have the functionality to add events to a calendar on the event site. Please also add this function to the order email below "URL". Leave one line of space below "URL".
2. Show "Add to calendar Google · Outlook · iCal · Yahoo". On clicking the service provide the desired option.
3. Ensure the text is translatable.

### PR DESCRIPTION
**Pull request by pc-beast**
Fixes #8280 
image

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/templates/email/ticket_purchased.html [hunk 1]
```diff
@@ -16,6 +16,11 @@
 <br/>{{ _('Location') }}: {{ order.event.event_location_status }}
 <br/>{{ _('URL') }}: {{ order.event.site_link }}
 <br/>
+<br/>{{ _('Add to calender') }}:    <a href="https://calendar.google.com/calendar/render?action=TEMPLATE&dates={{ order.event.starts_at.strftime('%Y%m%dT%H%M%') }}/{{ order.event.ends_at.strftime('%Y%m%dT%H%M%') }}&text={{ order.event.name }}&location={{ order.event.normalized_location }}&ctz={{ order.event.timezone }}&details={{ order.event.description }}">Google Calender</a>&bullet;
```
Comment (codedsun) [inline app/templates/email/ticket_purchased.html:19]: ~Google Calender~ Google Calendar

**Review by codedsun — CHANGES_REQUESTED** [review]

File: app/templates/email/ticket_purchased.html (hunk 1 above)
Comment (codedsun) [inline app/templates/email/ticket_purchased.html:19]: Add to Calendar

**Comment by mariobehling** [comment]
Thank you. Please add a space behind each option as below:
Google • Calendar • iCal • Yahoo • Outlook

**Comment by mariobehling** [comment]
Also only keep one empty line above "Best regards".

**Comment by pc-beast** [comment]
image

**Review by mariobehling — APPROVED** [review]

**Review by codedsun — APPROVED** [review]

**Pull request merged**